src/main/scala/org/tikv/bulkload/TiRegionSplitPartitionerV2.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 TiKV Project Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tikv.bulkload
+
+import org.apache.spark.Partitioner
+import org.tikv.common.key.Key
+import org.tikv.common.region.TiRegion
+
+class TiRegionSplitPartitionerV2(orderedRegions: List[TiRegion])
```

Comment:
rename to `RegionPartitioner`?

Reply:
Fixed in commit [[BulkLoad: fix README.md and keys must be added in order exception](https://github.com/tikv/migration/pull/1/commits/3c0520a1d4aefee81148474a77b47ff0fcacc6da)]